front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/messages.ts

```diff
@@ -128,6 +142,52 @@ export default defineMessages({
     id: 'app.containers.AdminPage.SettingsPage.twoRowLayout',
     defaultMessage: 'Two rows',
   },
+  twoRowBannerTooltip: {
+    id: 'app.containers.AdminPage.SettingsPage.twoRowBannerTooltip',
+    defaultMessage:
+      'This banner is in particular useful which images that don’t work well with text from the title, subtitle or button. These items will be pushed below the banner. More info on the recommended image usage can be found on our { link }.',
```

Comment:
Shouldn't it be `with` instead of `which` in `useful which images`?

Also, maybe it makes sense to ask @jinjagit to check all the copies?

Reply:
Yup, should be `...useful with images...`

Reply:
Alright, thanks, will change 👍